README.md

```diff
@@ -70,7 +77,16 @@ cd app
 npm start
 ```
 
-This will open http://localhost:3000 in your browser.
+This will open http://localhost:3000 in your browser. By default, it will use the production backend endpoint. 
+
+To test against the backend running locally, make this change in `app/src/features/editor/hooks/useCompile.tsx`:
+
+```diff
+-    const server_uri = 'https://api.sway-playground.org/compile';
+-    // const server_uri = 'http://0.0.0.0:8080/compile';
++    // const server_uri = 'https://api.sway-playground.org/compile';
++    const server_uri = 'http://0.0.0.0:8080/compile';
+```
```

Comment:
I think would be cool to add a check if by the `NODE_ENV` and on `development` just use the local compiler.

Reply:
Yes, I have a TODO for that, I'll do it in a separate PR